src/components/text-input/TextInput.scss

```diff
@@ -1,55 +1,105 @@
 @import '../../styles/variables';
+@import '../../styles/inputs';
 
-/**************************************
- * Text Input
- **************************************/
+.bdl-TextInput-input {
```

Comment:
This is not scoped inside `.bdl-TextInput-container` so that this class can be used on its own if needed. For some strangling html `<input>`.